View the observing request: https://observe.lco.global/requestgroups/1058776/
View the observing request: https://observe.lco.global/requestgroups/1058777/
View the observing request: https://observe.lco.global/requestgroups/1058778/
View the observing request: https://observe.lco.global/requestgroups/1058779/
View the observing request: https://observe.lco.global/requestgroups/1058780/
View the observing request: https://observe.lco.global/requestgroups/1058781/
View the observing request: https://observe.lco.global/requestgroups/1058782/
View the observing request: https://observe.lco.global/requestgroups/1058783/
View the observing request: https://observe.lco.global/requestgroups/1058784/
View the observing request: https://observe.lco.global/requestgroups/1058785/
View the observing request: https://observe.lco.global/requestgroups/1058786/
View the observing request: https://observe.lco.global/requestgroups/1058787/
Request failed: b'{"requests":[{"non_field_errors":["According to the constraints of the request, the target is visible for a maximum of 0.15 hours within the time window. This is less than the duration of your request 0.52 hours. Consider expanding the time window or loosening the airmass or lunar separation constraints."]}]}'
Traceback (most recent call last):
  File "tmp.py", line 26, in <module>
    raise exc
  File "tmp.py", line 23, in <module>
    response.raise_for_status()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://observe.lco.global/api/requestgroups/
View the observing request: https://observe.lco.global/requestgroups/1058788/
Request failed: b'{"requests":[{"non_field_errors":["According to the constraints of the request, the target is never visible within the time window. Check that the target is in the nighttime sky. Consider modifying the time window or loosening the airmass or lunar separation constraints. If the target is non sidereal, double check that the provided elements are correct."]}]}'
Traceback (most recent call last):
  File "tmp.py", line 26, in <module>
    raise exc
  File "tmp.py", line 23, in <module>
    response.raise_for_status()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://observe.lco.global/api/requestgroups/
Request failed: b'{"requests":[{"non_field_errors":["According to the constraints of the request, the target is never visible within the time window. Check that the target is in the nighttime sky. Consider modifying the time window or loosening the airmass or lunar separation constraints. If the target is non sidereal, double check that the provided elements are correct."]}]}'
Traceback (most recent call last):
  File "tmp.py", line 25, in <module>
    raise exc
  File "tmp.py", line 22, in <module>
    response.raise_for_status()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://observe.lco.global/api/requestgroups/
Request failed: b'{"requests":[{"non_field_errors":["According to the constraints of the request, the target is never visible within the time window. Check that the target is in the nighttime sky. Consider modifying the time window or loosening the airmass or lunar separation constraints. If the target is non sidereal, double check that the provided elements are correct."]}]}'
Traceback (most recent call last):
  File "tmp.py", line 25, in <module>
    raise exc
  File "tmp.py", line 22, in <module>
    response.raise_for_status()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://observe.lco.global/api/requestgroups/
Request failed: b'{"requests":[{"non_field_errors":["According to the constraints of the request, the target is never visible within the time window. Check that the target is in the nighttime sky. Consider modifying the time window or loosening the airmass or lunar separation constraints. If the target is non sidereal, double check that the provided elements are correct."]}]}'
Traceback (most recent call last):
  File "tmp.py", line 25, in <module>
    raise exc
  File "tmp.py", line 22, in <module>
    response.raise_for_status()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://observe.lco.global/api/requestgroups/
Request failed: b'{"requests":[{"non_field_errors":["According to the constraints of the request, the target is never visible within the time window. Check that the target is in the nighttime sky. Consider modifying the time window or loosening the airmass or lunar separation constraints. If the target is non sidereal, double check that the provided elements are correct."]}]}'
Traceback (most recent call last):
  File "tmp.py", line 25, in <module>
    raise exc
  File "tmp.py", line 22, in <module>
    response.raise_for_status()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://observe.lco.global/api/requestgroups/
View the observing request: https://observe.lco.global/requestgroups/1058789/
View the observing request: https://observe.lco.global/requestgroups/1058790/
View the observing request: https://observe.lco.global/requestgroups/1058791/
View the observing request: https://observe.lco.global/requestgroups/1058792/
View the observing request: https://observe.lco.global/requestgroups/1058793/
View the observing request: https://observe.lco.global/requestgroups/1058794/
View the observing request: https://observe.lco.global/requestgroups/1058795/
View the observing request: https://observe.lco.global/requestgroups/1058796/
View the observing request: https://observe.lco.global/requestgroups/1058797/
View the observing request: https://observe.lco.global/requestgroups/1058798/
View the observing request: https://observe.lco.global/requestgroups/1058799/
View the observing request: https://observe.lco.global/requestgroups/1058800/
View the observing request: https://observe.lco.global/requestgroups/1058801/
View the observing request: https://observe.lco.global/requestgroups/1058802/
View the observing request: https://observe.lco.global/requestgroups/1058803/
View the observing request: https://observe.lco.global/requestgroups/1058804/
View the observing request: https://observe.lco.global/requestgroups/1058805/
